### PR TITLE
Two small bug fixes for customer support tickets

### PR DESCRIPTION
**Pull request by doanac**

### CODE REVIEW

File: subcommands/targets/tag.go [hunk 1]
```diff
@@ -41,11 +41,13 @@ func init() {
...
-		unique[i] = item
-		items[item] = true
+	for _, item := range a {
+		if ok := items[item]; !ok {
+			unique = append(unique, item)
+			items[item] = true
+		}
 	}
 	for _, item := range b {
 		if ok := items[item]; !ok {
```
Comment (vkhoroz) [inline subcommands/targets/tag.go:53]: I feel like that second loop below should also have the `items[item] = true` logic to filter duplicates in it, shouldn't it.

Then, I'd aggregate the two into an outer loop like this:
```
for i, lst := 0, a; i < 2; i, lst = i+1, b {
	for _, item := range lst {
		if ok := items[item]; !ok {
			unique = append(unique, item)
			items[item] = true
		}
	}
}
```

Comment (vkhoroz) [reply subcommands/targets/tag.go:53]: FWIW it looks like a good candidate to use https://pkg.go.dev/github.com/mpvl/unique#Strings and https://pkg.go.dev/github.com/mpvl/unique#StringsAreUnique.

**Review by vkhoroz — APPROVED** [review]
LGTM

File: subcommands/targets/tag.go (hunk 1 above)
Comment (doanac) [reply subcommands/targets/tag.go]: i wasn't wanting to pull in another library, look at the code license,etc. But your loop idea looks great. force pushed

**Pull request merged**